data/group_stage.json

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
I would suggest to use more informative keys in the structure like wins, draws, losses etc